UI/web-src/ngMango/directives/setPointValue.js

```diff
@@ -217,6 +217,7 @@ function setPointValue(PointValueController, maTranslate, $q, $injector, maDialo
                 }, e => null);
             } else {
                 this.result = this.point.setValueResult(value);
+                this.close()
```

Comment:
- The callback needs to be called for the case above in the if statement too when confirmSet is true
- Only trigger the callback if the promise resolves successfully e.g. `this.result.promise.then(() => this.onSet())`